make/conf/jib-profiles.js

```diff
@@ -447,6 +447,15 @@ var getJibProfilesProfiles = function (input, common, data) {
                 "SETFILE=/usr/bin/SetFile"),
         },
 
+        "macosx-aarch64": {
+            target_os: "macosx",
+            target_cpu: "aarch64",
+            dependencies: ["devkit", "gtest"],
+            configure_args: concat(common.configure_args_64bit, "--with-zlib=system",
+                "--with-macosx-version-max=11.00.00",
+                "SETFILE=/usr/bin/SetFile"),
```

Comment:
"SetFile" from the devkit still does not work on the latest macOS?

Reply:
I'm a bit surprised at this myself. If it is needed, I will accept this patch, but then I think we need to fix that properly and submit a follow-up patch. 

I also note that on macos-x64 we add `--enable-compatible-cds-alignment` as well. I wonder if we should not do that here as well..? (Although I must admit I did not fully understand when and how the CDS alignment rules applied.)

Reply:
I don't think we need to set SETFILE anymore. This used to be needed because the SetFile binary in older Xcode versions was 32 bit and Catalina no longer supports 32 bit apps. This should definitely not be a problem on aarch64.

Reply:
I don't think we need the compatible cds alignment here. This was added for x64 mainly to make it run correctly in rosetta IIRC. Macos aarch64 binaries will most likely run well in their native environment without this.

Reply:
I'll drop the SETFILE override - I removed it, ran a test job and everything seems to work just as expected.

I agree with Erik that the CDS alignment shouldn't be an issue here.